pydecred/vsp.py

```diff
@@ -140,6 +150,8 @@ def __init__(self, url, apiKey):
         # The signingAddress (also called a votingAddress in other contexts) is
         # the P2SH 1-of-2 multi-sig address that spends SSTX outputs.
         self.signingAddress = None
+        self.isAccountless = isAccountless
+        self.ID = ID
```

Comment:
using an incremented id to track pools, rather than the api key, as accountless has none